internal/integrations/golang/gobinary.go

```diff
@@ -39,9 +38,9 @@ type GoBinary struct {
 var UseBuildKitForBuilding = knobs.Bool("golang_use_buildkit", "If set to true, buildkit is used for building, instead of a ko-style builder.", false)
 
 func (gb GoBinary) BuildImage(ctx context.Context, env pkggraph.SealedContext, conf build.Configuration) (compute.Computable[oci.Image], error) {
-	if testing.UseNamespaceBuildCluster || buildkit.BuildOnNamespaceCloud.Get(env.Configuration()) || UseBuildKitForBuilding.Get(env.Configuration()) {
-		return buildUsingBuildkit(ctx, env, gb, conf)
-	}
+	// if testing.UseNamespaceBuildCluster || buildkit.BuildOnNamespaceCloud.Get(env.Configuration()) || UseBuildKitForBuilding.Get(env.Configuration()) {
```

Comment:
this got accidentally commented out? or intended?

Reply:
Uh, sorry, good catch. Not intended.